README.rst

```diff
@@ -1,22 +1,42 @@
 Kickstart Test Documentation
 ****************************
 
+Kickstart tests are one way of testing the Anaconda Installer, by running an automated installation based on a kickstart file and checking the results.
+
 :Authors:
    Chris Lumens <clumens@redhat.com>
    Martin Kolman <mkolman@redhat.com>
```

Comment:
This list is missing Martin Pitt, I think?

(Also not sure about @rvykydal's earlier contributions.)